release.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+if [[ -n $(git status -s) ]]
+then
+  echo git working directory is not clean
+  exit 1
+fi
+
+versions=(${1//./ })
+
+tag=finntech/node
+onbuild_tag="$tag:onbuild"
+test_tag="$tag:test"
+test_onbuild_tag="$test_tag-onbuild"
+
+major=${versions[0]}
+minor=${versions[1]}
+
+patch_and_revision=(${versions[2]//-/ })
+
+patch=${patch_and_revision[0]}
+revision=${patch_and_revision[1]}
+
+tag_major="$tag:$major"
+tag_minor="$tag_major.$minor"
+tag_patch="$tag_minor.$patch"
+onbuild_tag_major="$onbuild_tag-$major"
+onbuild_tag_minor="$onbuild_tag_major.$minor"
+onbuild_tag_patch="$onbuild_tag_minor.$patch"
+test_tag_major="$test_tag-$major"
+test_tag_minor="$test_tag_major.$minor"
+test_tag_patch="$test_tag_minor.$patch"
+test_onbuild_tag_major="$test_onbuild_tag-$major"
+test_onbuild_tag_minor="$test_onbuild_tag_major.$minor"
+test_onbuild_tag_patch="$test_onbuild_tag_minor.$patch"
+
+if [[ -n "$revision" ]];
+  then
+    tag_major="$tag_major-$revision"
+    tag_minor="$tag_minor-$revision"
+    tag_patch="$tag_patch-$revision"
+    onbuild_tag_major="$onbuild_tag_major-$revision"
+    onbuild_tag_minor="$onbuild_tag_minor-$revision"
+    onbuild_tag_patch="$onbuild_tag_patch-$revision"
+fi;
```

Comment:
kinda inconsistent indentation from the previous if-then-fi, not that it actually matters